doc/faq.rst

```diff
@@ -212,14 +220,8 @@ Using Spyder
 
    Finally, you need to connect Spyder to this environment by changing Spyder’s default Python interpreter. To do this, click the name of the current environment in the status bar, and then click :guilabel:`Change default environment in Preferences`.
 
-   .. image:: /images/faq/faq-change-environment.png
```

Comment:
If this image is no longer needed, please remove it.

Reply:
This is being used somewhere else